.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+php:
+  - 7.1
```

Comment:
why do we use only 7.1? It should be really easy to add all versions we support, is there any reason not to?